website/docs/installation/evaluation.md

```diff
@@ -15,19 +14,39 @@ Run this command to start FerretDB with PostgreSQL + DocumentDB extension:
 docker run -d --rm --name ferretdb -p 27017:27017 --platform linux/amd64 ghcr.io/ferretdb/ferretdb-eval:2
 ```
 
-This command will start a container with FerretDB, pre-packaged PostgreSQL with DocumentDB extension, and MongoDB Shell for quick testing and experiments.
+This command will start a container that includes FerretDB, pre-packaged PostgreSQL with DocumentDB extension, and MongoDB Shell for quick testing and experiments.
 
-However, it is unsuitable for production use cases because it keeps all data inside and loses it on shutdown.
+:::caution
+It is unsuitable for production use cases because it keeps all data inside and loses it on shutdown.
 See other installation guides for instructions
 that don't have those problems.
+:::
 
 With that container running, you can:
 
-- Connect to it with any MongoDB client application using MongoDB URI `mongodb://username:password@127.0.0.1:27017/`.
-- Connect to it using MongoDB Shell by just running `mongosh`.
-  If you don't have it installed locally, you can run `docker exec -it ferretdb mongosh`.
-- For PostgreSQL, connect to it by running `docker exec -it ferretdb psql -U username postgres`.
+- Connect to it with any MongoDB client application using this MongoDB URI:
 
-In the [next step](usage/concepts.md), we will show you how to perform basic CRUD operations on FerretDB.
+  ```sh
+  mongodb://username:password@127.0.0.1:27017/
+  ```
+
+- Or connect to it using MongoDB Shell by just running `mongosh` if you have it installed already.
+  If you don't have it installed locally, you can run:
+
+  ```sh
+  docker exec -it ferretdb mongosh
```

Comment:
Doesn't evaluation image have credentials baked in? Shouldn't it be `docker exec -it ferretdb mongosh 'mongodb://username:password@127.0.0.1:27017/'`. Please double check